src/main/fc/init.c

```diff
@@ -538,7 +538,7 @@ void init(void)
     busSwitchInit();
 #endif
 
-#if defined(USE_UART)
+#if defined(USE_UART) && !defined(SIMULATOR_BUILD)
```

Comment:
There is stub `uartPinConfigure` for `SIMULATOR_BUILD`

Reply:
We should either remove this, or remove the stub.